dbt/profiles.yml

```diff
@@ -15,17 +16,16 @@ athena:
       type: athena
       s3_staging_dir: s3://ccao-dbt-athena-ci-us-east-1/results/
       s3_data_dir: s3://ccao-dbt-athena-ci-us-east-1/data/
+      s3_data_naming: schema_table
       region_name: us-east-1
       schema: dbt-test
       database: awsdatacatalog
-      # Prefix all generated data by schema, so that we can delete it when the
-      # PR is merged
-      s3_data_naming: schema_table
       threads: 5
     prod:
       type: athena
       s3_staging_dir: s3://ccao-athena-results-us-east-1/
-      s3_data_dir: s3://ccao-athena-data-us-east-1/
+      s3_data_dir: s3://ccao-athena-ctas-us-east-1/
+      s3_data_naming: schema_table
```

Comment:
**question (blocking):** Will merging this branch overwrite the existing production tables? If so, we may want to be cautious about a merge until the `dist_pin_to_pin` issue is resolved.

Reply:
It will! See my note in [this thread above](https://github.com/ccao-data/data-architecture/pull/111/files#r1321844067) for a suggestion of how we can handle this.